### DISCUSSION TITLE
Link a user's asset count directly to their list of assigned assets

### DISCUSSION THREAD

**Discussion by wildeep** [snipe/snipe-it#15000] [Ideas]
![image](https://github.com/snipe/snipe-it/assets/892364/d5834bb4-f88e-4ebe-a6ef-30dabf6aa865)

My primary use-case for Snipe-IT is "What assets are assigned to the user named X?"

So far I've been satisfying that question by clicking on 'users', then using the search field to enter a portion of the user's name. The results appear very quickly. For each user's row, I can see a display of numbers for how many assets, licenses, etc. are assigned.

When I see these numbers, particularly the assets number, I instinctively want to click it and go directly to a list of that user's assigned assets.

Instead, I'm forced to click the user's name, then find the link for the assets tab, and _then_ finally see that same list. More mousing, more clicks.

I'm proposing that the counts for assets (and possibly the other counts) be directly linked to the User>Assets tab.

**Comment by marcusmoore** [comment]
Good idea! I just created a PR for this  #15001

**Reply by wildeep** [reply]
Rock on, marcusmoore! This is the first time I've ever made a feature request for an open-source project. To see that it was considered, implemented, and pulled within 24 hours is **blowing my mind.**

**Reply by marcusmoore** [reply]
Thanks wildeep! It can't always work that way but I'm glad your first experience was a good one 😄
